housingResidentInformationAPI/serverless.yml

```diff
@@ -7,15 +7,15 @@ provider:
   region: eu-west-2
 
 package:
-  artifact: ./bin/release/netcoreapp3.1/uh-resident-information-api.zip
+  artifact: ./bin/release/netcoreapp3.1/housing-resident-information-api.zip
 
 functions:
-  uhResidentInformationApi:
+  housingResidentInformationApi:
     name: ${self:service}-${self:provider.stage}
-    handler: UHResidentInformationAPI::UHResidentInformationAPI.LambdaEntryPoint::FunctionHandlerAsync
+    handler: housingResidentInformationAPI::housingResidentInformationAPI.LambdaEntryPoint::FunctionHandlerAsync
     role: lambdaExecutionRole
     environment:
-      CONNECTION_STRING: Host=${ssm:/uh-api/${self:provider.stage}/postgres-hostname};Port=${ssm:/uh-api/${self:provider.stage}/postgres-port};Database=uh_mirror;Username=${ssm:/uh-api/${self:provider.stage}/postgres-username};Password=${ssm:/uh-api/${self:provider.stage}/postgres-password}
+      CONNECTION_STRING: Host=${ssm:housing-api/${self:provider.stage}/postgres-hostname};Port=${ssm:housing-api/${self:provider.stage}/postgres-port};Database=housing_mirror;Username=${ssm:housing-api/${self:provider.stage}/postgres-username};Password=${ssm:housing-api/${self:provider.stage}/postgres-password}
```

Comment:
We would need to rename these parameters in AWS too.

I think `Database=uh_mirror` would stay the same. Since we're still connecting to the UH database.

Reply:
We're missing out the `/` that comes after the `ssm:`.

Reply:
Just FYI: changing this will mean changing the names of all the parameters in parameters upon merge as well, for development, staging and production. I would suggest adding them as new parameters, then merging and after it has been pushed to production removing the old parameters.